### PR TITLE
Add Styling

### PR DESCRIPTION
**Pull request by ahzamir**
### In this pull request, I have added some styling to the app:

-  I have added some classes to my elements.
-  I have used my App.css file to add some styles to the elements.
-  I have added styles according to the Bookstore design in [Zeplin](https://zeplin.io/).

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -3,30 +3,49 @@ import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { removeBook } from '../redux/books/books';
 
-const Book = ({ title, author, id }) => {
+const Book = ({
+  title, author, id, category,
+}) => {
   const dispatch = useDispatch();
```
Comment (elyor-doniyorov) [inline src/components/Book.js:9]: - [ ] Your app's current design does not match to the given wireframe. Make sure that all elements in design are added. Kindly take a look at the below screenshots to see differences and fix them 👇 
- Your design 👇 
Screenshot (282)
- Original design 👇 
originale

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @ahzamir, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
✅ There are no linter errors
✅ Descriptive Readme file


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @ahzamir 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
- Well done making the requested changes

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahzamir** [comment]
@elyor-doniyorov  @MahmoudBakr23  Thank you very much for your kind review and feedback. 👍😊

**Pull request merged**